cv.md

```diff
@@ -0,0 +1,43 @@
+# Resume
```

Comment:
Looks like this tasks isn't ready yet :smile: no problem. Please, change the name `John, Beginner Web Masta` to something real.

Reply:
Ok, changed name.